website/src/app/page.tsx

```diff
@@ -106,43 +108,46 @@ export default function Page() {
       <section className="border-t border-neutral-200 bg-gradient-to-b from-white via-neutral-50 to-white py-24">
         <div className="mx-4 flex flex-col justify-center items-center">
           <h2 className="text-center justify-center mb-4 text-4xl tracking-tight font-bold text-neutral-900 ">
-            A modern alternative to legacy VPNs
+            Least-privileged access to your most valuable resources in just a
+            few minutes.
           </h2>
         </div>
 
         {/* Feature section 1 */}
         <div className="gap-8 items-center py-8 px-4 mx-auto max-w-screen-xl lg:grid lg:grid-cols-2 xl:gap-16 sm:py-16 lg:px-6 ">
           <div>
             <h4 className="mb-8 text-lg font-semibold tracking-tight text-primary-450 ">
-              SIMPLE TO MANAGE
+              EFFORTLESS SETUP
             </h4>
-            <h3 className="text-2xl font-semibold tracking-tight text-neutral-900">
-              Streamline workflows.
-              <br />
-              Reduce total cost of ownership.
-            </h3>
             <p className="text-xl text-neutral-800 my-4">
-              Legacy VPNs are cumbersome to manage and take weeks to configure
-              correctly. Firezone takes minutes to deploy and the Web GUI makes
-              managing secure access effortless for admins.
+              Replace your legacy VPN with a modern zero trust solution.
+              Firezone supports the workflows you're already familiar with, so
+              you can get started in minutes and incrementally adopt zero trust
+              over time.
             </p>
             <ul role="list" className="my-6 lg:mb-0 space-y-4">
               <li className="flex space-x-2.5">
                 <HiCheck className="flex-shrink-0 w-5 h-5 text-neutral-900 font-bold " />
                 <span className="leading-tight text-lg text-neutral-900 ">
-                  Integrate any identity provider to enforce 2FA / MFA
+                  Create a site
```

Comment:
Worth discussing (aka it's not optimal/best-practice), but I'd suggest linking "Site" to the docs for the first 6-12 months https://firezone.dev/kb/Site